src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -1199,6 +1199,12 @@ ImmutableArray<AttributeData> IMethodSymbol.GetReturnTypeAttributes()
         /// </summary>
         internal virtual void AddSynthesizedReturnTypeAttributes(ref ArrayBuilder<SynthesizedAttributeData> attributes)
         {
+            if (this.ReturnsByRefReadonly)
+            {
+                // PROTOTYPE(readonlyRefs) it is optional now as it will be generated in the next PR
+                var constructor = WellKnownMember.System_Runtime_CompilerServices_ReadOnlyAttribute__ctor;
+                AddSynthesizedAttribute(ref attributes, this.DeclaringCompilation.TrySynthesizeAttribute(constructor, isOptionalUse: true));
```

Comment:
A helper method is useful, even if the code is one line, if each instance has to duplicate some interesting value, in this case `isOptionalUse: true`. I don't feel strongly about this though given that the value will be dropped in a subsequent PR.

Is the `constructor` local necessary though?